File: onlineshopui/src/app/core/types/dtos/supplier.dto.ts
/**
 * Represents a product supplier with contact information.
 * Used in product responses and supplier management operations.
 */
export type SupplierDto = {
  id: string;
  name: string;
  description: string;
  contactEmail: string;
  contactPhone: string;
  address: string;
};

File: onlineshopapi/src/main/java/msg/onlineshopapi/dto/SupplierDto.java
package msg.onlineshopapi.dto;

import lombok.*;

import java.util.UUID;

/**
 * Represents a product supplier with contact information.
 * Used in product responses and supplier management operations.
 */
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SupplierDto {
    private UUID id;
    private String name;
    private String description;
    private String contactEmail;
    private String contactPhone;
    private String address;
}
